common/views/components/Paginator/Paginator.js

```diff
@@ -89,20 +98,32 @@ const Paginator = ({
     <Fragment>
       <Space
         h={{ size: 'm', properties: ['margin-right'] }}
```

Comment:
Could we just add the class to the space component? Why do we need a new wrapper?